### DISCUSSION TITLE
关于中外合作办学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#40] [Ideas]
> 转自群友

刚才别处有人问我中外合作办学的事情，分享一下我了解到的情况。最有用的是[教育部中外合作办学监管平台](http://www.crs.jsj.edu.cn)可用于查询办学详细情况。

## 缺点
学费高，少则两万多则十万。每年都有考生没看清楚误报了中外合作专业，之后才发现难以支付学费。

学长反映，部分合作办学项目教学内容与外国院校授课内容接轨，脱离考研需求，或大三大四课多影响考研，对考研不友好，可能不利于本科毕业后在国内深造。也有些合作办学项目的课程与国内基本一致。各学校的情况各异。

## 优点
分数线低于正常录取。
可能有外教授课或外语授课
部分项目可能有机会到外国就读，例如 2+2 指的是本科阶段在国内外各读两年。4+0 则是一直在国内
部分项目可能有外国院校发放的学位证及成绩单，有利于留学

具体办学内容，合作院校情况，发放证书参见[教育部中外合作办学监管平台](http://www.crs.jsj.edu.cn)

关于详细安排，如本科期间是否必须出国赴外方院校，学位证发放是否有特殊前提（例如必须出国才能发外方学位证）等事宜，需浏览各高校官网刊登的招生章程。

我也正在填志愿，这些都是听说的，汇总了一下目前我所了解到的情况。

**Comment by schlafen0901** [comment]
中外的狗来了，首先感觉大多数中外合作课都不是一般地多，想要大学课余时间多一点可以直接跑了，然后中外不是学托福的就是学小语种的，学小语种的课会非常多！相当于你本身专业课加上外语系专业课都要上！不想学小语种的可以跑了，然后要了解国外院校的学费吧，像欧美的学校不是一般贵，家境不太好的真的别报，接着就是真的要了解学院的出国规划之类的，比方说有的是进去了语言达标就能出去，有的是要通过外国教授的入学测试才能出去，有的是全院排位前50%才能出去，当然也有学校不出去，要是你想去看看国外的世界那么中外欢迎你，要是你想国内考研的话中外学习之余你要自己努力自己卷，因为以我为例，我的理工专业课都是英文上的，英文教材上的东西可能和中文不太一样，还有全英文试卷的难度也会比中文低，要是你要国内考研你必须往难里面卷自己学起来，另外你要是想国内考研你上中外其实也没有意义……它虽然分数低但是一般都是不让转到别的学院去的（会有例外，也会有别的学院来中外的），反正我觉得你来中外之前你要想清楚你之后的规划吧，是国内考研还是申请国外的研究生？是想要以后体制内工作还是去外企或者直接定居到国外呢？想清楚了就不要后悔啦。

**Comment by schlafen0901** [comment]
补充：就算是全英文的试卷一样很难！也不会比中文那边的简单太多！一样做不出！现在是凌晨，我还在为了期末考试而焦虑！妈的！反正来中外不可以摆烂！中外真的不是随便混混就能混出国的！卷起来！都给我卷起来！妈的！（阴暗地爬行）（扭曲）然后关于中外合作的师资，这个每个学校都不一样，我个人是遇到了很多很好很会教的老师，大一上学期下学期的两位高数老师真的超级会教！是能让本数学废柴轻松搞懂高数的那种，上课也非常有趣，根本不会睡着，听我说，谢谢你……（）然后还有一点就是有可能正常专业的学生会歧视中外合作的，毕竟低分进来的嘛，不过这个我到现在还没遇到公开表达这个观点的其他学院的同学（）这个就是和个人的三观教养有关系……然后中外还有好处就是因为课实在是很多所以有些无聊但必须要上的水课就免掉了，课表全是干货！保证你可以每天都学得很爽（bushi）以上是我个人上了中外合作一年的经历，不能代表所有的中外合作办学，仅供参考（）期末周本大学牲精神状态不太行想到什么就说什么，家人们看看就好（）

**Comment by TommyWang08** [comment]
某末流双一流的中外合作，来说说自己的身边统计学
缺点: 
1. 报这个专业多少会考虑出国的意向，所以班里有几个申申基本是逃不掉的，当然我也不知道非中外合作的专业申申多不多哈
2. 可能比你非中外合作的专业的分没低多少，但在别人眼里你就是地主家的傻儿子，但凡学校有资源多向你倾斜一点就等着在贴吧被挂上天龙人的帽子批判
3. 考研竞争力相较于其他专业确实会小，毕竟课程表上还有一大半的外方课程

优点
1. 虽然不绝对，但不得不承认家教确实与一个家庭的经济情况有关联，而报中外合作的大多都是家里有一定经济实力的，再加上成绩的限制，身边出现啥软的概率大大降低。我班里三十个人除了有个冲浪玩梗大师有点恶心人以外，别的大多数都很正常。当然，我也不知道非中外合作专业的情况哈。
2. 学校确实会有部分资源倾斜，像是教学楼宿舍啥的基本都是数一数二，不过毕竟你也付了这么多钱嘛。
3. 部分考试难度低，像是高数大物这些课程的考试，只要你看的懂全英文的卷子，难度确实会比别的院低。但外方课程就不一样了，会有quiz但不会占大分，期末基本都是pbl作业以小组的形式进行考核的，所以说对一定要找到好的队友，如果队友给力就和他敲定后续学期继续抱团别放他跑，对我这种社恐极其不友好，不主动抱团基本就是漏到后面和摆烂仔组队了。

聊聊别的点
关于英语: 我是从南方往北方考的，南北的英语方面确实有差异，像我英语本来就是后腿学科，大学还摆烂，但在班里英语成绩依旧中游。特别是高考有听力与口语考试的，英语这块确实是有优势。

关于课程: 每天一节的英语课肯定是跑不了了，课程基本都是国外外教一节国内助教一节轮流上，当然也有纯外教+助教辅助的。上课听得懂吗？懂，又不完全懂。好好上英语课加上问问助教理解课程肯定是没问题，但要知道外教不是什么新闻联播主持人，有些印度或者非裔外教的口语实在是不敢恭维。当然有些本校的老师也会怀着执念给你用英语上课，那口音只能说是人外有人。

关于出国: 我是20级，想必大火没少听当时国外的烂事，导致我们班二十八个人中只有两个有明确的出国意向，别的要么摇摆要么(就和我一样)准备留在国内。说到出国方面其实中外合作的外方大学基本都一般，如果真的考虑出国发展，基本当作跳板存在的。而且你的外方课程成绩在国外是受到认可的，可以直接去尝试申请别的学校，所以如果考虑出国基本都会朝着更高的目标考虑而不是选择中外合作的学校。

**Reply by L-zee** [reply]
悉院的？

**Comment by yuhouyinren** [comment]
想问一下哪些中外合作办学是会在毕业证上加个（中外合作）的，哪些是不会的。正好卡在较差211的差专业上很尴尬，想上合作办学又很怕这一点被歧视

**Reply by renzhuchu** [reply]
直接call目标院校的招生办问，最好精确到那个专业所在学院的招生办
